nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/Username.java

```diff
@@ -1,6 +1,9 @@
 package no.sikt.nva.nvi.common.service.model;
 
 import static java.util.Objects.isNull;
+import static nva.commons.core.StringUtils.isBlank;
+
+import no.sikt.nva.nvi.common.exceptions.ValidationException;
```

Comment:
Does this override `toString`, if it can be overridden, just return the value and make the class final